spec/RecapSpec.js

```diff
@@ -133,6 +133,42 @@ describe('The Recap export module', function() {
     });
   });
 
+  describe('uploadDocket, uploadAttachmentMenu, uploadDocument', function() {
+    var existingFormData;
+    beforeEach(function() {
+      setupChromeSpy();
+      existingFormData = window.FormData;
+      window.FormData = FormDataFake;
+    });
+    
+    afterEach(function() {
+      window.FormData = existingFormData;
+      removeChromeSpy();
+    });
+
+    var bytes = new Uint8Array([100, 100, 200, 200, 300]);
+
+    it('all send debug=false in AJAX request', function() {
+      expected = new FormDataFake();
+      expected.append('debug', false);
+
+      recap.uploadDocket(court, pacer_case_id, html, 'DOCKET', function() {});
+      var actualData = jasmine.Ajax.requests.mostRecent().data();
+      expect(actualData).toEqual(jasmine.objectContaining(expected));
```

Comment:
I thought we didn't want to use `objectContaining()` in a case like this when strict equality. would work?

Also, should we not be changing the `objectContaining()`s from my commit to `jasmine.any(String)` or `jasmine.any(Boolean)` or whatnot?

Reply:
Right, we don't need objectContaining() here because we just want to check one value. Updated.

As far as objectContaining, there's no inherent harm in using that (except that if the methods started sending extra parameters we wouldn't catch it). We don't actually want to use jasmine.any() because that's only for cases, like the nonce you mentioned, where there's some value and all we know about it is that it is a String, we can't know the value to test for strict equality.